char-sheet/src/components/NotesTable.tsx

```diff
@@ -30,7 +37,18 @@ function NotesTable({service, editable=EditMode.Live}: {
                   className='textarea textarea-bordered leading-tight w-full scrollbar scrollbar-neutral p-2'
```

Comment:
The input is auto-resizing, but you can still manually resize it. Maybe set `resize-none` to prevent manual resizing.